client/reader/components/header/style.scss

```diff
@@ -28,14 +28,8 @@ $component-icon-padding: 6px;
 	&.is-support-user-session {
 		background-color: var( --dashboard-header__support-background-color );
 	}
-}
-
-.dashboard-header-bar-title > span {
-	padding-inline: $grid-unit-15;
-}
 
-.dashboard-header-bar-title {
-	@include break-medium {
-		flex-shrink: 0;
+	.dashboard-secondary-menu__item:not(.is-active) {
```

Comment:
In https://github.com/Automattic/wp-calypso/pull/107665 my suggestion is to use styles from MSD directly and if that's the case then maybe we should add this style to MSD styles file.